wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_streamfield.scss

```diff
@@ -272,57 +298,83 @@ li.sequence-member {
         transform: translate3d(0, 0, 0) scale(1);
         visibility: visible;
         opacity: 1;
-        padding: 1em 1em 0;
+        padding: 1em 2em 1em;
+        display: flex;
+        flex-flow: row wrap;
         overflow: hidden;
     }
 
     li {
-        @include column(3);
-        padding-bottom: $grid-gutter-width;
-        padding-left: 0;
-        padding-right: 0;
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: column;
+        float: none;
+        width: 8em;
+        padding: 1em 0;
+        text-align: left;
+        margin: 1em;
+    }
 
-        &:nth-child(4n+1) {
-            clear: left;
-        }
+    h3 {
+        padding: 0.5em 0;
+        margin: 2em 3em 0;
+        border-bottom: 1px solid $color-menu-text;
+        padding-bottom: 1em;
```

Comment:
There is a lot of spacing definitions going on here. Could you refactor some of these to use variables so it's easier to see what is going on?